### ISSUE TITLE
Prepare for S2105R

### ISSUE THREAD

**Issue by samreid** [phetsims/forces-and-motion-basics#149]
From https://github.com/phetsims/tasks/issues/424 @ariel-phet suggested Forces and Motion: Basics. Its a popular sim, the friction changes and art changes would be nice.

@samreid suggested to check in with @jessegreenberg to make sure accessibility is in a good place before bringing this to production for S2015R.

@jessegreenberg said:

> Accessibility is in active development for forces-and-motion-basics. Net force screen is getting some revamped accessibility designs after an initial round of student testing. Accessible content for the remaining screens are being implemented for the first time now.

> However, references in code to accessibility changes should be safely hidden in blocks starting with this.accessibleContent = { ..., never being called without query parameter ?accessibility .

> So FAMB could be ready for redeploy.

> I could also continue accessibility in a branch, which is what I have started doing for other sims.

@samreid said: 

> Thanks @jessegreenberg, let's keep everything in master for now and we can re-evaluate if issues arise.

**Comment by samreid** [comment]
@ariel-phet said the priority of this issue should be: like a background task, no rush on it, start chipping away at open issues, etc.  My current other obligations include circuit construction kit basics, bending light, phet-io and common code issues.

**Comment by samreid** [comment]
I reviewed all open issues and marked 19 as part of the S2015R work.

**Comment by samreid** [comment]
I looked into https://github.com/phetsims/forces-and-motion-basics/issues/142 and reassigned to @ariel-phet for feedback.

**Comment by samreid** [comment]
I fixed https://github.com/phetsims/forces-and-motion-basics/issues/141 and reassigned to @ariel-phet for confirmation.

**Comment by samreid** [comment]
@ariel-phet I'm focusing heavily on PhET-iO then CCK in the near future, can someone else help with this issue?

**Comment by ariel-phet** [comment]
@jessegreenberg I am going to have you take over FAMB issues for redeploy since you are now quite familiar with this sim. FAMB should be prioritized after Trig Tour.

**Comment by jessegreenberg** [comment]
We are ready for a release branch for FAMB.  Once https://github.com/phetsims/forces-and-motion-basics/issues/171 is done on the release branch, we can possibly pull in old translations for FAMB for the translation utility, but I am not sure how that process works and where old translations are located.  I will look into that and ask some other developers.

**Comment by jessegreenberg** [comment]
2.0.0 will use common code SHAS from http://www.colorado.edu/physics/phet/dev/html/energy-skate-park-basics/1.1.0-rc.2/dependencies.json, which were approved for publication, but with the updated chipper SHA https://github.com/phetsims/chipper/commit/55c02ee94221a6b2cfa022295d60d68cdd37fef3 for percent symbols in translated strings.

**Comment by jessegreenberg** [comment]
#171 has been closed.

**Comment by jessegreenberg** [comment]
@jbphet talked me through the process of replacing the changed keys in babel and he ran a script that quickly removed all of the keys in babel string files that are not being used in the HTML5.  I think we are all ready for an RC.

**Comment by jessegreenberg** [comment]
Testing task for 2.0.0-rc.1 created here: https://github.com/phetsims/tasks/issues/583

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/tasks/issues/583 passed for the redeploy with the exception of https://github.com/phetsims/forces-and-motion-basics/issues/191, the observed common code memory leak from https://github.com/phetsims/tasks/issues/546 since this RC uses the current [stable set of SHAs](https://github.com/phetsims/tasks/issues/476#issuecomment-214479205) which do not include the memory leak patch.  A new RC will be deployed with common code at master to fix #191, and to test master so we can include the leak patch to the remaining redeploys.

**Comment by jessegreenberg** [comment]
New SHAs passed through testing and no issues came from testing of 2.0.0-rc.2.  With #192 fixed, 2.0.0 is ready to be deployed.

**Comment by jessegreenberg** [comment]
2.0.0 successfully deployed to the site!

**Comment by samreid** [comment]
Hooray, nice work and congratulations @jessegreenberg!

**Comment by jessegreenberg** [comment]
Thank you @samreid!  I think this issue can now be closed.
